CHANGELOG.md

```diff
@@ -20,8 +20,7 @@
 
 - Brian Beckerle ([@brainbicycle](https://github.com/brainbicycle))
 
----
-
+--
```

Comment:
```suggestion
---

```

We should probably leave this how it was given that this document is automatically updated.